frontend/src/contracts.ts

```diff
@@ -250,7 +254,7 @@ export async function setUpContracts(web3: Web3): Promise<Contracts> {
   }
 
   const pvpContracts: PvPContracts = {};
-  if(featureFlagPvP){
+  if(!featureFlagPvP){
```

Comment:
Why is this change here? If PvP is on -> initialize contract, if not -> don't do it, it will crash the app. Or am I wrong here?

Reply:
Nice catch, I was playing with it earlier I guess.